src/main/java/org/spongepowered/asm/mixin/transformer/InnerClassGenerator.java

```diff
@@ -158,10 +159,29 @@ void accept(final ClassVisitor classVisitor) throws ClassNotFoundException, IOEx
             classNode.accept(classVisitor);
             this.loadCounter++;
         }
-        
+
+        /**
+         * Used to remap fields which have been renamed while being applied to
+         * the target class or by an implementation of IRemapper.
+         *
+         * @see org.spongepowered.asm.mixin.extensibility.IRemapper
+         */
+        @Override
+        public String mapFieldName(String owner, String name, String descriptor) {
+            if (this.ownerName.equalsIgnoreCase(owner)) {
```

Comment:
Whilst this is just copied from `mapMethodName`, class names in Java (and the JVM in general) are case sensitive so just a normal `equals` might be a better idea.

Reply:
Probably a good change.